products/bio-formats/downloads/index.html

```diff
@@ -4,7 +4,7 @@
 ---     
         <div class="callout large primary" id="bg-image-bio-formats">
             <div class="row column text-center">
-                <h1>Bio-Formats [5.x.x] Downloads</h1>
+                <h1>Bio-Formats 5.5.0 Downloads</h1>
                 <p>[ a blurb about the Downloads ]</p>
```

Comment:
We can just delete it if you don't think having extra text there is informative/needed